app/controllers/wellcome_controller.rb

```diff
@@ -14,4 +16,8 @@ def admin_user_redirect
       render :index
     end
   end
+
+  def user_archived_redirect
+    current_user.archived ? (render :archived) : (render :index)
```

Comment:
we have `redirect` in action name but then we never actually redirect in it, fix naming accordingly